e2e/rdctl.e2e.spec.ts

```diff
@@ -855,6 +873,16 @@ test.describe('Command server', () => {
           expect(result.stderr).toEqual('');
         });
       });
+
+      test('complains about options not intended for current platform', async() => {
+        // See https://github.com/microsoft/playwright/issues/7036 for the explanation
+        // why I'm not doing `test.describe(... set up the array ... test.each(array)...
```

Comment:
So, in summary: playwright doesn't support `test.each`? That seems like a more concise and clear comment…